apps/rebar/src/rebar_env.erl

```diff
@@ -58,7 +58,8 @@ create_env(State, Opts) ->
         {"ERLANG_TARGET",           rebar_api:get_arch()}
     ],
     EInterfaceVars = create_erl_interface_env(),
-    lists:append([EnvVars, EInterfaceVars]).
+    ConfigVars = rebar_state:get(State, env, []),
```

Comment:
we're gonna need a far more descriptive name for this option, especially since `rebar_env` is called exclusively for hooks right now, and that `env` can also refer to application or release environment values.

A name like `shell_hooks_env` would likely be the most descriptive and narrow definition. `hooks_env` could work but could be confusing for provider hooks rather than shell hooks. `env_hooks` would fit the overall config pattern but is sort of a stretch as well.

Either way, we need something narrower and more specific than just `env` here.

Reply:
I have no problem with `shell_hooks_env`